app/views/content/github/_review_checklist.erb

```diff
@@ -25,11 +24,13 @@
 - [ ] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
 - [ ] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
 - [ ] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
-- [ ] **Automated tests:** Are there automated tests or manual steps described so that the function of the software can be verified?
+- [ ] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
 - [ ] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support
 
 ### Software paper
 
-- [ ] **Authors:** Does the `paper.md` file include a list of authors with their affiliations?
+- [ ] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
 - [ ] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
+- [ ] **State of the art:** Are key relevant or alternative approaches cited and briefly discussed?
```

Comment:
I'm not sure `State of the art` reflects our [submission guidelines](https://joss.readthedocs.io/en/latest/submitting.html#submission-requirements). What about `**Significant contribution to open source: ** Is this software a significant contribution to the available open source software in this research area?`

Reply:
Actually you are raising a different point. What I meant by this "State of the art" point is whether or not the authors describe the current state of the art in their paper. Currently our guidelines say there should be: "A list of key references including a link to the software archive". First of all we no longer should ask for a link to the software archive (will be added automatically), second of all we do not just want a "list of key references" we want them to write a nice paragraph where they summarize what I called the "state of the art", or more specifically "Are key relevant or alternative approaches cited and briefly discussed". So this statement was aimed at replacing the references statement and aims to hint more at the quality of writing. 
Your point on significance of the work in general can also be added but perhaps that is captured by the statement of need stuff anyway. Thoughts?

Reply:
Any thoughts on this. Let me know if I can help or if we should chat about this.